packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -37,10 +36,15 @@ const ConditionalLink: FC< { active: boolean } & LinkProps > = ( { active, ...pr
 	return <span { ...props }></span>;
 };
 
-export const HelpCenterContactPage: FC = () => {
+type HelpCenterContactPageProps = {
+	hideHeaders?: boolean;
```

Comment:
I think this (named differently) should also be used to disable the ai quick responses in the contact form (after you choose the contact method). Since you've already chatted with Wapuu, no need to get additional AI answers. 

See #81344 for the changes needed to disable them.

Reply:
I don't understand what you mean, would you mind elaborating? I'm just not seeing why that should be disabled. I think we want to test a regular flow as with everything was set up normally, but just having Wapuu as main actor.